packages/client/src/revenue/Revenue.ts

```diff
@@ -0,0 +1,37 @@
+import { GraphQLClient } from 'graphql-request';
+
+import { LensConfig } from '../consts/config';
+import {
+  ProfileFollowRevenueQueryRequest,
+  ProfilePublicationRevenueQueryRequest,
+  PublicationRevenueQueryRequest,
+} from '../graphql/types.generated';
+import { getSdk, Sdk } from './graphql/revenue.generated';
+
+export class Revenue {
+  private readonly sdk: Sdk;
+
+  constructor(config: LensConfig) {
+    const client = new GraphQLClient(config.environment.gqlEndpoint);
+
+    this.sdk = getSdk(client);
+  }
+
+  async profilePublicationRevenue(request: ProfilePublicationRevenueQueryRequest) {
+    const result = await this.sdk.ProfilePublicationRevenue({ request });
+
+    return result.data.result;
+  }
+
+  async profileFollowRevenue(request: ProfileFollowRevenueQueryRequest) {
+    const result = await this.sdk.ProfileFollowRevenue({ request });
+
+    return result.data.result;
+  }
+
+  async publicationRevenue(request: PublicationRevenueQueryRequest) {
```

Comment:
shall we add return types on the methods here to make it obvious what is returned?